core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1747,6 +1748,12 @@ object SnappySession extends Logging {
   private[this] val ID = new AtomicInteger(0)
   private[sql] val ExecutionKey = "EXECUTION"
 
+  lazy val isEnterpriseEdition: Boolean = {
+    GemFireVersion.getInstance(Class.forName("io.snappydata.gemxd.SnappyDataVersion"),
```

Comment:
Yes, but that is in snappy-cluster module which snappy-core cannot access, right?